src/openrct2/object/Object.cpp

```diff
@@ -329,6 +329,47 @@ std::unique_ptr<IStream> ObjectAsset::GetStream() const
     return {};
 }
 
+std::string VersionString(Version version)
+{
+    utf8 buff[VERSION_STRING_MAX_LENGTH];
+    return String::Format(
+        buff, VERSION_STRING_MAX_LENGTH, "%i.%i.%i", std::get<0>(version), std::get<1>(version), std::get<2>(version));
+}
```

Comment:
Why not make this a method of `Version`?

Reply:
The type is just a using of tuple.

Reply:
oh fair enough